config.json

```diff
@@ -618,7 +618,38 @@
       "name": "Type Conversion"
     }
   ],
-  "key_features": [],
+  "key_features": [
+    {
+      icon: "web",
+      title: "Popular for Web and Mobile Development",
+      content: "Dart has the Flutter library, which can be used to quickly build web and smartphone applications."
+    },
+    {
+      icon: "cross-platform",
+      title: "Instant iOS and Android Development",
+      content: "Dart/Flutter can be converted into iOS's Swift language and Android's Kotlin language, making mobile development fast and efficient."
+    },
+    {
+      icon: "community",
+      title: "Full Documentation",
+      contents: "Dart's documentation and learning materials are constantly updated by its developers, making it easy to look up information on the language."
+    },
+    {
+      icon: "community",
+      title: "Countless Packages",
+      contents: "Dart allows users to build and publish their own packages for public use, which can be accessed from Dart directly."
```

Comment:
This may be a good spot to add `pub.dev` where all the Dart packages live.

Reply:
We have pretty tight character limits. The key features aren't really intended for directing people to resources; it's really just to entice people to try the track.